graph/pull_v2.go

```diff
@@ -76,6 +76,9 @@ func (p *v2Puller) pullV2Repository(tag string) (err error) {
 	broadcaster.Add(p.config.OutStream)
 	if found {
 		// Another pull of the same repository is already taking place; just wait for it to finish
+		if dryRun {
+			broadcaster.Write(p.sf.FormatStatus(tag, "!!! Another pull of the same image is already running, dry-run is not possible unless you restart the Docker daemon !!!"))
```

Comment:
This should write to `p.config.OutStream` rather than `broadcaster`; otherwise the message will go to all clients that are watching this pull.

I'd also suggest changing the message to remove the `!!!` characters, and also remove "unless you restart the Docker daemon" (because the user only needs to wait for the pull to finish).